File: mod_13_devops_e_cloud/topicos/projetos/modulo2_programacao/java/ListaTarefas.java
import java.util.ArrayList;
import java.util.Scanner;

public class ListaTarefas {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<String> tarefas = new ArrayList<>();

        while (true) {
            System.out.print("Digite uma tarefa (ou 'sair' para encerrar): ");
            String tarefa = sc.nextLine();
            if (tarefa.equalsIgnoreCase("sair")) break;
            tarefas.add(tarefa);
        }

        System.out.println("\nSuas tarefas:");
        for (int i = 0; i < tarefas.size(); i++) {
            System.out.println((i+1) + ". " + tarefas.get(i));
        }
        sc.close();
    }
}

File: mod_13_devops_e_cloud/topicos/projetos/modulo2_programacao/python/lista_tarefas.py
tarefas = []

while True:
    tarefa = input("Digite uma tarefa (ou 'sair' para encerrar): ")
    if tarefa.lower() == "sair":
        break
    tarefas.append(tarefa)

print("\nSuas tarefas:")
for i, t in enumerate(tarefas, start=1):
    print(f"{i}. {t}")
